src/stateless/TypeaheadMultiselect.vue

```diff
@@ -50,11 +51,13 @@ export default {
             }
         },
         selectItem (item) {
+            this.$root.$emit('tracking-event', { type: 'input', label: this.trackName, trigger: 'select-item' })
             this.$emit('input', this.value.concat([item]))
             this.text = ''
             this.$refs.typeahead.focus()
         },
         removeItem (item) {
+            this.$root.$emit('tracking-event', { type: 'input', label: this.trackName, trigger: 'remove-item' })
```

Comment:
Maybe add `item.id` in data.

Reply:
same as above

Reply:
it was intentional decision not to include this since in the majority of the event we just track that something was clicked or focused. We do however send data in some components where DI team asked for it, but for this one, we don't want to track it since it can leak personal info like emails on Receipients list. (GDPR)